src/Microsoft.AspNet.Mvc.Core/Routing/UrlHelper.cs

```diff
@@ -46,7 +46,7 @@ public virtual string Action(UrlActionContext actionContext)
                 throw new ArgumentNullException(nameof(actionContext));
             }
 
-            var valuesDictionary = PropertyHelper.ObjectToDictionary(actionContext.Values);
```

Comment:
Could we delete `ObjectToDictionary` or are there other places its still being used?